doc/source/whatsnew/v0.17.0.txt

```diff
@@ -182,6 +181,30 @@ Other enhancements
    s.drop_duplicates(keep=False)
 
 
+- ``concat`` will now use existing Series names if provided (:issue:`10698`).
+
+  Previous Behavior:
+
+  .. code-block:: python
+
+     In [1]: foo = pd.Series([1,2], name='foo')
+     In [2]: bar = pd.Series([1,2])
+     In [3]: baz = pd.Series([4,5])
+     In [4] pd.concat([foo, bar, baz], 1)
+     Out[4]:
+           0  1  2
+        0  1  1  4
+        1  2  2  5
+
+  New Behavior:
+
+  .. ipython:: python
+
+    foo = pd.Series([1,2], name='foo')
```

Comment:
just create all of these in a separate python block above previous behavior. Then no need to show them twice. (obviously show the results in the previous behavior section as a code-block then in an ipython block in new behavior).